CODEX-DAF-RENAME-NOTICE.md

```
@@ -0,0 +1,158 @@
+deleted paths.
+
+The port is verified faithful. Every gate reports the same numbers as before
+the rename, to the digit: crest sweep -0.141 / -0.540 / +0.615 held out /
+-0.709 dB, burst rate fitted RMS 0.561 dB, detector weighting shape RMS
+0.097 dB, short-event release worst 1.386 dB.
+
+## Commands, updated
+
+Core suite (fast, this is the one that carries the Opto gates):
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)